vue/components/ui/molecules/table/table.vue

```diff
@@ -340,7 +353,7 @@ export const Table = {
         },
         classes() {
             const base = {
-                alignment: this.alignment === "left" ? "text-align-left" : ""
+                alignment: this.alignment === "right" ? "text-align-right" : ""

```

Comment:
@Benedito0 please revert this part. Even though it's in the "javascript" part, it's actually related with the UI and with the fact that the `table` here has a different default alignment that the table in RIPE White.

Reply:
@Benedito0 this is still unresolved...

Reply:
Sorry, foi sem querer o request